client/reader/tag-stream/header.jsx

```diff
@@ -15,6 +14,10 @@ import { getTagImages } from 'calypso/state/reader/tags/images/selectors';
 const TAG_HEADER_WIDTH = 800;
 const TAG_HEADER_HEIGHT = 140;
 
+function sample( array ) {
```

Comment:
The `getTagImages` selector used in this file can return `undefined` but using `defaultProps` to set `tagImages` to an empty array should account for that 👍🏻